fw/http_parser.c

```diff
@@ -5689,7 +5689,7 @@ __h2_req_parse_authority(TfwHttpReq *req, unsigned char *data, size_t len,
 	__FSM_STATE(Req_I_A_v6) {
 		/* See Req_UriAuthorityIPv6 processing. */
 		if (likely(isxdigit(c) || c == ':'))
-			__FSM_H2_I_MOVE_n_flag(Req_I_A_v6, 1, TFW_STR_VALUE);
+			__FSM_H2_I_MOVE_fixup(Req_I_A_v6, 1, TFW_STR_VALUE);
```

Comment:
That was the only usage of `__FSM_H2_I_MOVE_n_flag()`, so we can remove the macro for now.

The other usages of `__FSM_H2_I_MOVE_LAMBDA_n_flag()` are `__FSM_H2_I_MOVE_LAMBDA_n()` and `__FSM_H2_I_MOVE_n`. `__FSM_H2_I_MOVE_LAMBDA_n()` is used in only 2 places: `__h2_req_parse_authority()` in `Req_I_A` and it fixes up a chunk via lambda, and in `__h2_req_parse_cache_control()` without any fixups. `__FSM_H2_I_MOVE_n` is used in a lot of places and it's likely that there are no fixups at all - how the things work if we reach the end of data? Don't we have the same problem in all the places? It seems we just don't set the current TfwStr chunk length in all the cases or do we?